src/AppBundle/Event/EventInvitation.php

```diff
@@ -41,8 +42,69 @@ class EventInvitation
      */
     public $guests = [];
 
+    public static function createFromAdherent($adherent = null): self
+    {
+        $dto = new self();
+
+        if ($adherent instanceof Adherent) {
```

Comment:
Ok got why the if is needed, still the parameter should be type hinted with a nullable class:
```php
public static function createFromAdherent(?Adherent $adherent)
{
    // ...
}
```